cmd/expiration-mailer/main.go

```diff
@@ -349,6 +349,9 @@ type config struct {
 func main() {
 	configFile := flag.String("config", "", "File path to the configuration file for this service")
 	certLimit := flag.Int("cert_limit", 0, "Count of certificates to process per expiration period")
+	retryBase := flag.Duration("retryBase", 1*time.Second, "Base sleep duration between reconnect attempts")
+	retryMax := flag.Duration("retryMax", 30*60*time.Second, "Max sleep duration between reconnect attempts after exponential backoff")
+	retryAttempts := flag.Uint("retryAttempts", 100, "Max number of reconnects to attempt before quitting")
```

Comment:
I think we should call these specifically reconnect\*. Also, we should have only one of Max vs Attempts since they solve the same problem in different ways. My vote would be for Max. And maybe reduce it to 5m.